docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/02_Website_Configuration/README.md

```diff
@@ -1 +1,16 @@
-Web configuration.
\ No newline at end of file
+# Web configuration
+
+## Security: Enabling SSL
+
+*Note: SSL is not fully supported for the Data Querying Tool. Individual Loris instances should set up their own SSL for their couchApp hosting.*
+
+###Enable SSL module in apache
+`sudo a2enmod ssl`
+
+## Edit `/etc/apache2/sites-available/<project-name>`
+VirtualHost port: change to *:443 (ssl) instead of *:80 (http)
+Verify: SSL Engine On
+SSL certificate: ensure that you have an SSL certificate (contact us if you have questions)
+
```

Comment:
@driusan you might know this better than me but the instructions here do not seem to be generic at all and might not even for for our internal wiki.

I think we are better off presenting it as a "best practice" section in the developers instructions, just like the "make sure to backup your multi-million dollars of data" above.

Reply:
Changing the port to the the standard HTTP port to the standard SSL port seems pretty generic for apache, and I think the a2enmod is pretty standard apache, but I'm not sure how to interpret `SSL certificate: ensure that you have an SSL certificate (contact us if you have questions)`

Reply:
What I mean is, we (internally) don't enable SSL on a VM level if I'm not mistaken. I think IT does it on a server level (but again, i'm probably not the best person to ask) and it might be the case for many other institutions

Reply:
I'm not sure, but enabling SSL seems like a pretty generic thing to want to do, even if we do it at a different level internally.

Reply:
then I vote for a general best practice section